ntbs-service/Services/ValidationService.cs

```diff
@@ -147,19 +150,19 @@ public bool IsValid(string key)
             return ModelState()[key].Errors.Count == 0;
         }
 
-        public void TrySetAndValidateDateOnModel(object model, string key, FormattedDate formattedDate)
+        /// <param name="model"> The model on which date gets set </param>
+        /// <param name="modelKey"> Prefix for model state errors </param>
+        /// <param name="key"> Date key for model state errors </param>
+        /// <param name="formattedDate"> The FormattedDate to covert and set </param>
+        public void TrySetFormattedDate(object model, string modelKey, string key, FormattedDate formattedDate)
         {
-            string modelTypeName = model.GetType().Name;
-
             if (formattedDate.TryConvertToDateTime(out DateTime? convertedDob))
             {
                 model.GetType().GetProperty(key).SetValue(model, convertedDob);
-                pageModel.TryValidateModel(model, modelTypeName);
```

Comment:
Is that validation happening elsewhere now?

Reply:
Yes, all places that used this method called `TryValidateModel` themselves pretty much straight afterwards. I've checked that it stayed consistent between them

Reply:
Cool!